config.js.example

```diff
@@ -6,23 +6,23 @@ let settings = {
         server: process.env.IRC_SERVER || "irc.freenode.net",
         channel: process.env.IRC_CHANNEL || "",
         botName: process.env.IRC_BOT_NAME || "teleirc",
-        sendStickerEmoji: process.env.IRC_SEND_STICKER_EMOJI || true,
+        sendStickerEmoji: process.env.IRC_SEND_STICKER_EMOJI === 'true'|| true,
```

Comment:
Huge nitpick from a consistency standpoint, but this should probably be using either single quotes or double quotes and keeping it consistent (it uses single quotes for this and the two below it then switches to double quotes)

Reply:
I missed that one!